src/opu/processor/movement.py

```diff
@@ -86,4 +86,4 @@ def activate_motor(cortical_area, motor_id, speed_reference):
 
 def convert_motor_speed_to_twist_msg(motor_speed):
     twist_msg = round(map_value(motor_speed, 0, 20, 0, 10))
-    return -twist_msg
+    return twist_msg
```

Comment:
Are we still using the twist message for the motor or this is a placeholder function incase we revert back to the diff-driver?

Reply:
This is just being used to map motor values coming from FEAGI to suitable values for use in the simulation. I originally named it when we were using twist messages. I think it needs to be updated (both the name and the mapping range).